Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -45,6 +45,13 @@
                          *globalValidation
                          *hltvalidation)
 
+# fastsim: remove bits that depend on digis, tracker simhits or tracker rechits
+def _validation_fastSimMods(sequence):
+    sequence.remove(globaldigisanalyze)
+    sequence.remove(globalhitsanalyze)
+    sequence.remove(globalrechitsanalyze)
+eras.fastSim.toModify(validation,func=_validation_fastSimMods)
```

Comment:
@lveldere I have no objections either, just wanted to raise the point to @Dr15Jones.

@Dr15Jones Thanks, didn't think of `map`. With `toReplaceWith()`, would a good pattern be then the following?
```
eras.fastSim.toReplaceWith(someSequence, someSequence.copyAndExclude([b, d]))
```
May I also suggest to update the era documentation in https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCmsDriverEras? Thanks to the rapid progress in the past weeks it starts to feel a bit outdated, and I think up-to-date documentation will be essential to communicate the preferred patterns to non-framework developers that are starting to use the era mechanism now.